modules/ding_frontend/ding_frontend.module

```diff
@@ -6,6 +6,33 @@
 
 include_once 'ding_frontend.features.inc';
 
+function ding_frontend_permission() {
+  return array(
+    'administer fe promote' =>  array(
+      'title' => t('Administer promoting to frontpage'),
+    ),
+  );
+}
+
+/**
+ * Implements hook_menu().
+ */
+function ding_frontend_menu() {
+  $items = array();
+
+  $items['admin/config/ding/fe-promote'] = array(
+    'title' => 'Promote CT\'s to frontpage',
+    'description' => 'Promote several CT\' to frontpage.',
```

Comment:
This is more related to administering which content types support frontpage promotion.

Also: Please avoid abbreviations here.